app/code/Magento/Quote/Model/GuestCart/GuestShippingAddressManagement.php

```diff
@@ -6,7 +6,6 @@
 namespace Magento\Quote\Model\GuestCart;
 
 use Magento\Quote\Model\GuestCart\GuestShippingAddressManagementInterface;
-use Magento\Quote\Model\QuoteIdMask;
```

Comment:
We can't remove this line because `Magento\Quote\Model\QuoteIdMask` used in `assign` and `get` method without the fully qualified name